packages/muze-axis/src/cartesian-axis/simple-axis.js

```diff
@@ -299,9 +299,8 @@ export default class SimpleAxis {
     getLogicalSpace () {
         if (!this.logicalSpace()) {
```

Comment:
`this.logicalSpace()` is called two times here

Reply:
`logicalSpace` setter is called if it is not already set, else it is called as a getter whose value is returned